frontend/src/component/admin/groups/GroupsList/GroupCard/GroupCardAvatars/GroupPopover/GroupPopover.tsx

```diff
@@ -44,7 +44,7 @@ export const GroupPopover = ({
             }}
         >
             <StyledName>{user?.name || user?.username}</StyledName>
-            <div>{user?.email}</div>
+            <div>{user?.description || user?.email}</div>
```

Comment:
I'm a little unsure about this one: why are you changing the behavior of an existing component here? Will this have unintended consequences? Do we even use group descriptions in the new api? Isn't it just the group name?